### PR TITLE
Change type of 'severities' argument to enum in GetDebugMessageLogAMD

### LINKED ISSUE #516: GetDebugMessageLogAMD takes GLint* argument instead of GLenum*
The function `GetDebugMessageLogAMD` from `GL_AMD_debug_output` is defined as follows: 
```c
uint GetDebugMessageLogAMD(uint count,
                               GLsizei bufsize,
                               GLenum* categories,
                               GLuint* severities,
                               GLuint* ids,
                               GLsizei* lengths, 
                               GLchar* message);
```

Shouldn't the `severities` argument be of type `GLenum`? 
This would match every other definition of this function from other extensions.

As they are `typedef`ed as the same type is it possible to change this?

### PR DESCRIPTION
**Pull request by NogginBops**
Fixes #516 

Not sure if I should add this change to the change log of the extension text.
I also noticed now that the text later refers to `GetDebugMessageLogAMD` with the new signature, so the extension was inconsistent to begin with.

I'm also adding an extra commit where I change instances of `enum` to be `GLenum`, I can easily revert this if desired.

### CODE REVIEW

**Comment by oddhack** [comment]
LGTM.

File: extensions/AMD/AMD_debug_output.txt [hunk 1]
```diff
@@ -93,13 +93,13 @@ IP Status
 New Procedures and Functions
     
     void DebugMessageEnableAMD(enum category,
-                               enum severity,
+                               GLenum severity,
```
Comment (Perksey) [inline extensions/AMD/AMD_debug_output.txt:96]: Extension specifications omit prefixes for readability.

**Comment by NogginBops** [comment]
@oddhack reverted the `enum` to `GLenum` commit.

**Comment by NogginBops** [comment]
I can't figure out from your message if I should revert the changelog change too, but I'm guessing that there is something more as you said I should revert more than one thing. But I'm unsure of what I'm supposed to remove.

**Comment by oddhack** [comment]
> I can't figure out from your message if I should revert the changelog change too, but I'm guessing that there is something more as you said I should revert more than one thing. But I'm unsure of what I'm supposed to remove.

LGTM. I should not have used "revert" in the first sentence, all I meant was I had previously signed off and then replaced that comment after realizing the GL prefix needed to be dropped.

**Comment by oddhack** [comment]
@pdaniell-nv I think this is OK per discussion on #516? Also, if @rpatlega is a WG member from AMD, please let me know so I can add them to the Github team and @-ping them.

**Comment by rpatlega** [comment]
From AMD side, we are good this for merge.

**Comment by oddhack** [comment]
> From AMD side, we are good this for merge.

Thanks. I'll take that as Signed Off given @pdaniell-nv's comment. BTW I sent you an invite to the github team associated with the OpenGL repos.

**Pull request merged**